src/app/content/content.component.ts

```diff
@@ -34,12 +40,23 @@ export class ContentComponent implements OnInit {
         this.articleListData = this.filteredArticles;
         return;
       }
-      this.articleListData = this.filteredArticles.filter(
-        source =>
-          source.category.includes(pattern) ||
-          source.content.includes(pattern) ||
-          source.title.includes(pattern)
-      );
+      this.articleListData = this.filteredArticles.filter(source => {
+        if (source === null) {
+          console.log("Chutiyapa");
```

Comment:
Make sure you remove these `console.log` statements

Reply:
`Removed`